app/root/root.tsx

```diff
@@ -24,7 +25,7 @@ export default class RootComponent extends React.Component {
     hash: window.location.hash,
     path: window.location.pathname,
     search: new URLSearchParams(window.location.search),
-    denseMode: window.localStorage.getItem(denseModeKey) == denseModeValue || false
+    denseMode: denseModeKey in window.localStorage ? JSON.parse(window.localStorage.getItem(denseModeKey)) : window.buildbuddyConfig && window.buildbuddyConfig.default_to_dense_mode
```

Comment:
In order to maintain backwards compatibility of people's dense mode preference and the flexibility to add more modes - I think we should be able to keep the denseModeKey value as "VIEW_MODE" and maybe introduce another value `const comfyModeValue = "COMFY";`

Then we could do something like this:
```
denseMode: denseModeKey in window.localStorage ? (window.localStorage.getItem(denseModeKey) == denseModeValue)  : (window.buildbuddyConfig && window.buildbuddyConfig.default_to_dense_mode);
```

And below we could toggle between "DENSE" and "COMFY" instead of the true & false strings.

What do you think?

Reply:
Sounds good to me, will update shortly :)